brave/src/main/java/brave/Tracing.java

```diff
@@ -210,11 +208,9 @@ public Builder endpoint(zipkin2.Endpoint endpoint) {
     }
 
     /**
-     * Controls how spans are reported. Defaults to logging, but often an {@link AsyncReporter}
-     * which batches spans before sending to Zipkin.
-     *
-     * The {@link AsyncReporter} includes a {@link Sender}, which is a driver for transports like
-     * http, kafka and scribe.
+     * Controls how {@linkplain TraceContext#sampled() remote sampled} spans report to a {@code
+     * io.zipkin.reporter2:zipkin-reporter} destination. This input is usually a {@link
```

Comment:
Maybe an `<a>` tag linking to the module or something. Currently, it's hard to know this is a maven artifact.

Reply:
good idea